netmaster/k8snetwork/networkpolicy.go

```diff
@@ -0,0 +1,424 @@
+package networkpolicy
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/contiv/contivmodel/client"
+	"github.com/contiv/netplugin/utils/k8sutils"
+	"github.com/rchirakk/client-go/kubernetes"
```

Comment:
@rchirakk: Looks like the networkpolicy related changes is checked into the official client-go. Please check and move to official client-go.